NeoMathEngine/include/NeoMathEngine/NeoMathEngine.h

```diff
@@ -779,6 +789,11 @@ class NEOMATHENGINE_API IDnnEngine : public IBlasEngine {
 		const CConstFloatHandle& input, const CConstFloatHandle& outputDiff, const CFloatHandle& filterDiff,
 		const CFloatHandle* freeTermDiff ) = 0;
 
+	virtual CSmallMatricesMultiplyDesc* InitSmallMatricesMultiplyDesc( int firstHeight, int firstWidth, int secondWidth, int resultWidth,
```

Comment:
`trans2` goes before `trans1` in public interface - definitely should be fixed

Reply:
The order is specially selected, because `trans2` is more frequently changed than `trans1`.
Maybe should I better use one enum-argument instead of 3 bool-s?

Reply:
The problem is that in any other library (you can check MKL/OpenBLAS/MLAS) `OpA` goes before `OpB`. You don't have default values for them anyway so frequency doesn't matter. The reversed order is _extremely_ counter-intuitive.

Plus, in other NeoMathEngine funcs `secondRowSize` goes right after `secondWidth`. I think it's better here to do the same.

Reply:
fixed